### PR TITLE
signup page created

### PR DESCRIPTION
**Pull request by MayurChawla**
Signup page created

signup

### CODE REVIEW

File: Signup/Signup.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 2px solid var(--primary-color);
+    align-items: center;
+    background-color: white;
+}
+nav .text-input {
+    width: 50%;
+    padding: 0.5rem 1rem;
+}
```
Comment (omjadhav85) [inline Signup/Signup.css:21]: Most of this css is already added in your styles.css file. You don't need to rewrite it here instead you can just import that file

File: Signup/Signup.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import url(https://king-ui.netlify.app/styles.css);
+@import url(https://king-ui.netlify.app/Stylesheets/navbar.css);
+
+@import url(https://king-ui.netlify.app/Stylesheets/reset.css);
```
Comment (omjadhav85) [inline Signup/Signup.css:4]: I suggest use only single link for your CL import

**Review by omjadhav85 — APPROVED** [review]
Other than suggested changes looks good to me!

File: Signup/Signup.css (hunk 1 above)
Comment (MayurChawla) [reply Signup/Signup.css:21]: i have imported, wanted some custom changes

File: Signup/Signup.css (hunk 2 above)
Comment (MayurChawla) [reply Signup/Signup.css:4]: noted

**Comment by aurobindoGupta** [comment]
good work

**Pull request merged**